Admin's Manual.md

```diff
@@ -104,9 +104,22 @@ twitter:
     access_token_secret: XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 ```
 
-You can use both integrations together.
+### Using the RPC integration
 
-There are 3 environment variables you can set to quickly disable integrations on the command line:
+Devzat includes a built-in gRPC server. This is useful for building your own integration or using a third-party one.
+
+Documentation for using the gRPC API is available [here [TODO]](<TODO>).
```

Comment:
Alright so we need some documentation for people looking to use the API

Reply:
I just added some, take a look and let me know what you think! (it's at /plugin/README.md)

Reply:
Thank you so much!!